src/README.md

```diff
@@ -0,0 +1,42 @@
+![ThreadSharp banner](https://github.com/itsWindows11/ThreadSharp/blob/main/assets/banner.png?raw=true)
+
+## Documentation
+
+You can browse the documentation at [itswindows11.github.io/ThreadSharp](https://itswindows11.github.io/ThreadSharp/docs/).
+
+## Usage
+
+You can start by initializing a `ThreadsClient` with an access token:
+
+```c#
+ThreadsClient threadsClient = new ThreadsClient("access-token");
+```
+
+## Building the Code
+
+### Prerequisites
+
+Ensure you have following components:
+
+- [Git](https://git-scm.com/)
+- a .NET IDE of your choice, preferably [Visual Studio 2022](https://visualstudio.microsoft.com/vs/)
+- the .NET SDK
+
+### Git
+
+Clone the repository:
+
+```git
+git clone https://github.com/itsWindows11/ThreadSharp
+```
+
+### Build the project
+
+- Open `ThreadSharp.sln`.
+- Build `ThreadSharp.csproj` with `DEBUG` mode activated
+
+## License
+
+Copyright (c) 2024 `itsWindows11` & `ThreadSharp` contributors.
```

Comment:
```suggestion
Copyright (c) 2024 itsWindows11 & ThreadSharp contributors.
```
Code blocks aren't necessary for this.
P.S. it probably won't work on NuGet.

Reply:
They do work on NuGet.